conf/test_stitch.config

```diff
@@ -0,0 +1,34 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/phaseimpute -profile test_stitch,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name        = 'Minimal Stitch Test profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function using the tool STITCH'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = '2.GB'
+    max_time   = '1.h'
+
+    // Input data
+    input        = "${projectDir}/tests/csv/sample_bam.csv"
+    input_region = "${projectDir}/tests/csv/region.csv"
+
+    // Genome references
+    fasta  = "https://raw.githubusercontent.com/nf-core/test-datasets/phaseimpute/data/reference_genome/21_22/hs38DH.chr21_22.fa"
+    phased = true
+    panel  = "${projectDir}/tests/csv/panel.csv"
```

Comment:
Shouldn't we remove the panel as stitch normally doesn't need it ?

Reply:
When removing, it currently breaks because prepare panel includes `params.step == 'impute'`. This should be removed (but I'll have to modify `glimpse1` as well). 

```

    //
    // Prepare panel
    //
    if (params.step == 'impute' || params.step == 'panel_prep' || params.step == 'validate' || params.step == 'all') {
        // Check chr prefix and remove if necessary

```